File: clear-all/clear-all/src/main/java/com/example/clear_all/dto/response/FixtureMatchQuery.java
package com.example.clear_all.dto.response;

import lombok.Data;

import java.math.BigDecimal;


@Data
public class FixtureMatchQuery {
    private BigDecimal id;
    private String homeTeamName;
    private String awayTeamName;
    private String matchDateTime;
    private String status;
}

File: WebSport24hNews/Application/Query/Model/24hMatches/FixtureMatchQuery.cs
namespace WebSport24hNews.Application.Query.Model._24hMatches
{

    //lớp dto cho mỗi trận đấu trong lịch thi đấu
    public class FixtureMatchQuery
    {
        public decimal Id { get; set; }
        public string HomeTeamName { get; set; }
        public string AwayTeamName { get; set; }
        public string MatchDateTime { get; set; } // Định dạng chuỗi '25/05 22:00'
        public string Status { get; set; }
    }
}
